src/comm/aca_comm_mgr.cpp

```diff
@@ -446,10 +443,36 @@ void Aca_Comm_Manager::print_goal_state(GoalState parsed_struct)
             current_RouterConfiguration.subnet_routing_tables_size());
 
     for (int j = 0; j < current_RouterConfiguration.subnet_routing_tables_size(); j++) {
-      fprintf(stdout, "current_RouterConfiguration.subnet_routing_tables(%d): %s\n", j,
+      fprintf(stdout, "current_RouterConfiguration.subnet_routing_tables(%d).subnet_id: %s\n", j,
```

Comment:
We may have too many fprintf in this method, would slow down quite a bit in a stress test.

Reply:
that's only enabled with -d (debug) option during development. We will switch to better logging after we updated our logging engine #116